scripts/run_lighthouse_tests.py

```diff
@@ -190,29 +206,31 @@ def main(args=None):
     """Runs lighthouse checks and deletes reports."""
     parsed_args = _PARSER.parse_args(args=args)
 
-    if not (parsed_args.accessibility or parsed_args.performance):
-        _PARSER.error(
-            'Must pass either --accessibility or --performance flag')
+    if parsed_args.mode == 'accessibility':
```

Comment:
Can we use LIGHTHOUSE_MODE_ACCESSIBILITY etc. here instead (once you rename the constants)?